src/registrar/models/utility/generic_helper.py

```diff
@@ -59,6 +59,11 @@ def create_or_update_organization_type(self):
         one of the excluded types (FEDERAL, INTERSTATE, SCHOOL_DISTRICT), the
         organization_type is set to a corresponding election variant. Otherwise, it directly
         mirrors the generic_org_type value.
+
+        args:
+            force_update (bool): If an existing instance has no values to change,
+            try to update the organization_type field (or related fields) anyway.
+            This is done by invoking the new instance handler.
```

Comment:
Add: "Use to force org type to be updated to the correct value even if no other changes were made (including is_election)